ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ProfileClustersPage.tsx

```diff
@@ -47,6 +51,8 @@ function ProfileClustersPage() {
                     <Divider />
                     <ProfileClustersTable
                         currentDatetime={currentDatetime}
+                        pagination={pagination}
+                        profileClustersResultsCount={profileClusters?.totalCount ?? 0}
```

Comment:
Wow, another example of optional chaining and nullish coalescing. We might discuss in team meeting.